thirdparty/scripts/build_modin.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+
+# Cause the script to exit if a single command fails.
+set -e
+
+TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)/../
+MODIN_VERSION=0.2.2
+MODIN_REPO="https://github.com/modin-project/modin"
+(cd $TP_DIR/../python/ray/ && rm -rf modin && wget "$MODIN_REPO"/archive/v"$MODIN_VERSION".tar.gz && tar -xvzf v"$MODIN_VERSION".tar.gz && mv modin-"$MODIN_VERSION" modin && rm v"$MODIN_VERSION".tar.gz)
```

Comment:
Why is this one really long line? Why not put everything on separate lines?

Also, instead of `cd` use `pushd`/`popd`